src/resource/user/entities/user.entity.ts

```diff
@@ -1,8 +1,17 @@
 import { ObjectType, Field } from '@nestjs/graphql';
+import { Link } from '@src/resource/link/entities/link.entity';
```

Comment:
I noticed you're applying two importing ways in this project. Generally speaking, we should just use one to keep consistency and avoid confusion.

Reply:
Ahh, yeah, thanks!